modules/gapi/test/common/gapi_render_tests.cpp

```diff
@@ -0,0 +1,9 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
```

Comment:
Let's update?
```suggestion
// Copyright (C) 2019 Intel Corporation
```

Reply:
same in other places

Reply:
I think that could be aligned generally with some annual copyrights update.

Reply:
makes sense. consider resolved from my side

Reply:
makes sense, consider resolved from my side